tests/fingerprints_tests.py

```diff
@@ -23,63 +13,77 @@
     TopologicalTorsionFingerprint,
     ERGFingerprint,
 )
-from rdkit.Chem.rdReducedGraphs import GetErGFingerprint
+
+import rdkit.Chem.rdFingerprintGenerator as fpgens
+from scipy.sparse import csr_array
 
 
 @pytest.fixture
 def example_molecules():
     return [
+        "Oc1ncnc2c1sc1nc3ccccc3n12",
+        "CC1=CC(=C(c2cc(C)c(O)c(C(=O)O)c2)c2c(Cl)ccc(S(=O)(=O)O)c2Cl)C=C(C(=O)O)C1=O.[NaH]",
+        "Cc1ccc2nsnc2c1[N+](=O)[O-]",
         "COc1cccc(NC(=O)CC(=O)N2N=C(N(CCC#N)c3ccc(Cl)cc3)CC2c2ccccc2)c1",
         "CCN(CCO)CCNc1ccc(C)c2sc3ccccc3c(=O)c12",
         "Oc1ncnc2c1sc1nc3ccccc3n12",
         "CC1=CC(=C(c2cc(C)c(O)c(C(=O)O)c2)c2c(Cl)ccc(S(=O)(=O)O)c2Cl)C=C(C(=O)O)C1=O.[NaH]",
         "Cc1ccc2nsnc2c1[N+](=O)[O-]",
+        "COc1cccc(NC(=O)CC(=O)N2N=C(N(CCC#N)c3ccc(Cl)cc3)CC2c2ccccc2)c1",
+        "CCN(CCO)CCNc1ccc(C)c2sc3ccccc3c(=O)c12",
+        "Oc1ncnc2c1sc1nc3ccccc3n12",
+        "CC1=CC(=C(c2cc(C)c(O)c(C(=O)O)c2)c2c(Cl)ccc(S(=O)(=O)O)c2Cl)C=C(C(=O)O)C1=O.[NaH]",
+        "Cc1ccc2nsnc2c1[N+](=O)[O-]",
+        "COc1cccc(NC(=O)CC(=O)N2N=C(N(CCC#N)c3ccc(Cl)cc3)CC2c2ccccc2)c1",
+        "CCN(CCO)CCNc1ccc(C)c2sc3ccccc3c(=O)c12",
+        "Oc1ncnc2c1sc1nc3ccccc3n12",
+        "CC1=CC(=C(c2cc(C)c(O)c(C(=O)O)c2)c2c(Cl)ccc(S(=O)(=O)O)c2Cl)C=C(C(=O)O)C1=O.[NaH]",
     ]
 
 
-def test_morgan_fingerprint(example_molecules):
+def test_morgan_bit_fingerprint(example_molecules):
     X = example_molecules
+    X_for_rdkit = [Chem.MolFromSmiles(x) for x in X]
```

Comment:
If you make fixtures from this, it will allow you to just pass those to functions as arguments

Reply:
Done